.catalog-onboard-pipeline.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+offerings:
+  - name: deploy-arch-secrets-manager-private-cert-engine
+    kind: solution
+    catalog_id: 7df1e4ca-d54c-4fd0-82ce-3d13247308cd
+    offering_id: 571d2eb7-e416-40a4-aa95-be05e6155af8
+    variations:
+      - name: fully-configurable
+        mark_ready: true
+        install_type: fullstack
+        pre_validation: "tests/scripts/pre-validate.sh solutions/fully-configurable"
+        post_validation: "tests/scripts/post-validate.sh"
```

Comment:
why have you added `pre_validation` and `post_validation` ? There are no validation script needed for this DA?